test/testsuppressions.cpp

```diff
@@ -298,7 +298,7 @@ class TestSuppressions : public TestFixture {
                                         "    a++;\n"
                                         "}\n",
                                         "uninitvar:test.cpp"));
-        ASSERT_EQUALS("", errout.str());
```

Comment:
can you motivate this.

Reply:
See previous comment: https://github.com/danmar/cppcheck/pull/4882#issuecomment-1464175461